### ISSUE TITLE
Final Fantasy Crystal Chronicles - Ring of Fates - Anti-Piracy measure does not trigger if using --slot1-no8000prot

### ISSUE THREAD

**Issue by Sukotto-1999** [TASEmulators/desmume#547]
Like the title says, i could not trigger the anti-piracy in FFCC - Ring of Fates if using --slot1-no8000prot even if playing for 20 minutes. I'm trying to plan on document my research of the anti-piracy measure in FFCC - Ring of Fates. Maybe the protection is based on the game's DMA3 address and is not the CRC. Maybe the game does not relie on the retail copy protection of the cartridge. The anti-piracy measure in the game is, it should show the "Thank you for playing!!!" screen after 20 minutes of playing. Any help?

**Comment by Sukotto-1999** [comment]
Okay, i tried NO$GBA or old versions of DeSmuME. No avail. This anti piracy feature just made the news on various sites (search it up on Google) and patches are even made. This is the only anti-piracy measure in any DS game that i could not trigger outside of NDS-bootstrap

**Comment by zeromus** [comment]
SUCCESS. PERFECT EMULATOR FAILS TO TRIP AP.

what the hell do you even want us to do here? create deficient emulation that trips the AP?

**Comment by Sukotto-1999** [comment]
What? Only an option that trips, the copy protection? Well, how about a special cmd option that trips the AP, how about a new option that can be enabled via command line, how about "--slot1-no8000protimprov" that allows tripping the copy protection.

**Comment by Sukotto-1999** [comment]
you know what? I give up, i may have no choice to find old emulators until i found one that trips the copy protection. A user was reportedly using DeSmuME version 0.9.4 dev build.

**Comment by Sukotto-1999** [comment]
Hi then, zeromus. I found a way to trigger the game's anti-piracy in the DeSmuME emulator. I used Pokemonhacker's research dating from 2008-2009, and i hacked the game's memory based on the user's research, then i got the game's anti-piracy to trigger. I also found a way to trigger the thank you screen automatically. By the way, i made the following Action Replay codes, and we don't ever need a new option to trigger the copy protection.

The first code tricks the game into thinking it's being run on some flashcarts:
020FB594 00000111

The second code triggers the Thank you screen automatically (requires the first code to activate):
020FB659 00008012
